eventstream_rpc/include/aws/eventstreamrpc/EventStreamClient.h

```diff
@@ -299,23 +307,63 @@ namespace Aws
             ContinuationCallbackData *m_callbackData;
         };
 
+        /**
+         * A wrapper for event-stream-rpc client continuation.
+         */
         class AWS_EVENTSTREAMRPC_API ClientContinuation final
         {
           public:
+            /**
+             * Create a new continuation.
+             *
+             * @note continuation_option's callbacks will not be invoked, and nothing will be sent across
+             * the wire until Activate() is invoked.
+             * @param connection Connection on which open a new stream.
+             * @param continuationHandler A set of callbacks that will be invoked for continuation events.
+             * @param allocator Allocator to use.
+             */
             ClientContinuation(
                 ClientConnection *connection,
                 ClientContinuationHandler &continuationHandler,
                 Crt::Allocator *allocator) noexcept;
             ~ClientContinuation() noexcept;
+
+            /**
+             * Initiate a new client stream. Send new message for the new stream.
+             * @param operation Name for the operation to be invoked by the peer endpoint.
+             * @param headers Headers for the eventstream message.
+             * @param payload Payload for the eventstream message.
+             * @param messageType Message type for the message.
+             * @param messageFlags Bitmask of aws_event_stream_rpc_message_flag values.
+             * @param onMessageFlushCallback Callback to be invoked upon the message being flushed to the underlying
+             * transport.
+             * @return Future that will be resolved when the message has either been written to the wire or it fails.
+             */
             std::future<RpcError> Activate(
                 const Crt::String &operation,
                 const Crt::List<EventStreamHeader> &headers,
                 const Crt::Optional<Crt::ByteBuf> &payload,
                 MessageType messageType,
                 uint32_t messageFlags,
                 OnMessageFlushCallback onMessageFlushCallback) noexcept;
+
+            /**
+             * Check if the continuation has been closed.
+             * @return True if the continuation has been closed, false otherwise.
+             */
             bool IsClosed() noexcept;
             void Release() noexcept;
```

Comment:
Release probably needs a comment

Reply:
This method doesn't have implementation. I removed this method in my original attempt to document this module: https://github.com/aws/aws-iot-device-sdk-cpp-v2/pull/694/files#diff-d1487c1bd42c87e8c6d157086720cf4623bfaa5146fc1060072df42b1471cffcL318.
Eventually I decided to split the original PR into multiple: one for docs, one for cleaning up the code, one for fixing various issues. Because it started turn into unreviewable mess pretty quick.

Saying that, I don't have a strong opinion on the matter, so I can remove unimplemented methods (there are more than one) if you think it's the right thing to do here.

Reply:
That's fine then.